kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -0,0 +1,96 @@
+server {
+    listen 80;
+
+    gzip                     on;
+    gzip_min_length          1000;
+    gzip_proxied             expired no-cache no-store private auth;
+    gzip_types               text/plain text/css text/javascript application/xml;
+    client_max_body_size     {{ max_upload_size }};
+
+
+    location /download/ {
+        alias /srv/storage/;
+    }
+
+    location = /upload/ {
+
+        access_by_lua_block {
+            -- Module nginx big upload support only content-type: "application/x-www-form-urlencoded"
+            check_content_type_header=ngx.req.get_headers()["Content-Type"]
+            if check_content_type_header ~= "application/x-www-form-urlencoded" then
+                ngx.status=400
+                ngx.say('Only Content-Type: "application/x-www-form-urlencoded" is allowed!')
+                ngx.exit(400)
```

Comment:
The right response in this case is `HTTP 415 UNSUPPORTED MEDIA TYPE`.

Reply:
Done.

Reply:
You changed all three cases to 415. This is incorrect. Only the first one check media type. The other two should be HTTP 400.

Reply:
Done.